JS.md

```diff
@@ -266,7 +266,7 @@ WebAssembly Exported Functions have a `[[Call]](this, argValues)` method defined
  * Let `argTypes` be the list of value types defined by the signature of [[FunctionIndex]].
  * Let `args` be an empty list of coerced values.
  * For each value type `t` in `argTypes` and value `v` in `argValues`:
-   * Append [`ToWebAssemblyValue`](#towebassemblyvalue)`(v)` to `args`.
+   * Execute [`ToWebAssemblyValue`](#towebassemblyvalue)`(v)` in the [[Realm]] of the Exported Function and append the result to `args`.
```

Comment:
Since `ToJSValue` technically also depends on the current realm (for the prototype of the possibly-thrown `Error` object), I think we should say that the entire method body executes in [[Realm]].  Looking at the spec more (specifically 9.3 and 9.3.1), this is actually mandated, so we don't have a choice here.  Perhaps we could add, as a first bullet: "As \[required of all builtin function objects\](https://tc39.github.io/ecma262/#sec-built-in-function-objects), this procedure executes in the [[Realm]] of the callee Exported Function."